lib/miq_memcached.rb

```diff
@@ -2,6 +2,10 @@
 require 'linux_admin'
 
 module MiqMemcached
+  def self.server_address
+    ENV["MEMCACHED_SERVER"] || ::Settings.session.memcache_server
```

Comment:
Unfortunately, `MEMCACHED_SERVICE_NAME` is only "memcached".  It doesn't include the port.  I was trying to make this a more generic solution, like how Rails uses `DATABASE_URL`.